packages/cubejs-client-react/index.d.ts

```diff
@@ -178,7 +178,7 @@ declare module '@cubejs-client/react' {
     query?: Query;
   };
 
-  type QueryBuilderRenderProps = {
+  type QueryBuilderRenderProps = Partial<QueryRendererRenderProps> & {
```

Comment:
LGTM, but

![image](https://user-images.githubusercontent.com/572096/95741311-04989380-0cc9-11eb-8363-053e875d94f3.png)

To get `QueryRendererRenderProps` props inside `render` fn, we should pass `wrapWithQueryRenderer` as a prop to `QueryBuilder`.

I can suggest:

```typescript
  type QueryBuilderPropsWithWrap = QueryBuilderBaseProps & {
    wrapWithQueryRenderer: true;
    render: (renderProps: QueryRendererRenderProps & QueryBuilderRenderProps) => React.ReactNode;
  }

  type QueryBuilderPropsWithoutWrap = QueryBuilderBaseProps & {
    wrapWithQueryRenderer?: false;
    render: (renderProps: QueryBuilderRenderProps) => React.ReactNode;
  }
  
  type QueryBuilderProps = QueryBuilderPropsWithWrap | QueryBuilderPropsWithoutWrap;
```

But it looks likes over engineering